whatsappcrm_backend/products_and_services/models.py

```
@@ -137,6 +137,14 @@ class Meta:
         verbose_name = _("Product")
         verbose_name_plural = _("Products")
         ordering = ['name']
+        indexes = [
+            # Matches the public shop query: filter(is_active, published) ordered
+            # by name. Lets the storefront list be served straight from the index.
+            models.Index(
+                fields=['published', 'is_active', 'name'],
+                name='product_shop_listing_idx',
+            ),
+        ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)